src/content/en/updates/2017/12/chrome-64-media-updates.md

```diff
@@ -0,0 +1,349 @@
+3. Turn off internet connection.
+4. Click the "Play" button and enjoy the video!
+
+## Media preload defaults to "metadata" {: #media-preload-defaults-metadata }
+
+Matching other browsers implementations, Chrome desktop now sets the default
+preload value for `<video>` and `<audio>` elements to "metadata" in order to
+reduce bandwidth and resource usage. This new behaviour only applies to cases
+where no preload value is set. Note that the preload attribute's hint is
+discarded when a `MediaSource` is attached to the media element as the web site
+handles its own preload.
+
+In other words, `<video>` preload value is now "metadata" while `<video
+preload="auto">` preload value stays "auto". Give a try to the [official
+sample](https://googlechrome.github.io/samples/media/preload-metadata).
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/5CDvJkdxyQ8) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5682169347309568) &#124;
+[Chromium Bug](https://crbug.com/310450)
+
+## Unsupported playbackRate raises an exception {: #unsupported-playbackRate-raises-exception }
+
+Following an [HTML specification change], when media elements' playbackRate
+is set to a value not supported by Chrome (eg. negative value), a
+"NotSupportedError" DOMException is thrown in Chrome 63.
+
+```js
+const audio = document.querySelector('audio');
+try {
+  audio.playbackRate = -1;
+} catch(error) {
+  console.log(error.message); // Failed to set the playbackRate property
+}
+```
+
+For info, Chrome’s current implementation raises this exception when new
+playbackRate is either negative, less than 0.0625, or more than 16. Give a try
+to the [official
+sample](https://googlechrome.github.io/samples/media/playback-rate-exception)
+to see this in action.
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/a0tguvcZZyk) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5750156230131712) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=747082)
+
+## Background video track optimizations {: #background-video-track-optimizations }
+
+The chrome team is always trying to find new ways to improve battery life and
+Chrome 63 was no exception.
+
+If the video doesn't contain any audio tracks, the video will be automatically
+paused when played in the background in Chrome desktop. This is a follow-up
+from a similar change that was only applying to [MSE videos in Chrome 62].
+
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=663999)
+
+## Remove muting for extreme playbackRates {: #remove-muting-extreme-playbackrates }
+
+Before Chrome 64, sound was muted when playbackRate was below 0.5 and above 4
+as the quality degraded significantly. As Chrome has switched to a
+Waveform-Similarity-Overlap-Add (WSOLA) approach for quality degrading, sound
+doesn’t need to be muted anymore. It means you can play sound crazy slow and
+fast now.
```

Comment:
In this case, I'd use the adjective on both nouns: 'crazy slow and crazy fast'.

Reply:
Done.